apiserver/Makefile

```diff
@@ -164,13 +164,22 @@ docker-push: ## Push image for the api server.
 
 ##@ Deployment
 .PHONY: install
-install: kustomize ## Install the kuberay api server to the K8s cluster specified in ~/.kube/config.
-	cd deploy/local && $(KUSTOMIZE) edit set image kuberay/apiserver=$(IMG)
-	$(KUSTOMIZE) build deploy/local | kubectl create -f -
+install: kustomize docker-image load-image  ## Install the kuberay api server to the K8s cluster specified in ~/.kube/config.
+	cd deploy/local/insecure && $(KUSTOMIZE) edit set image kuberay/apiserver=$(IMG)
+	$(KUSTOMIZE) build deploy/local/insecure | kubectl create -f -
 
 .PHONY: uninstall
 uninstall: ## Install the kuberay api server to the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build deploy/local | kubectl delete -f -
+	$(KUSTOMIZE) build deploy/local/insecure | kubectl delete -f -
+
+.PHONY: install-secure
```

Comment:
Can you add a descriptiuon for the target? `## Instal.....`

Reply:
Done